magnolia/shared/src/test/scala-2.12-2.13/zio/config/magnolia/AutomaticConfigSpec.scala

```diff
@@ -26,13 +26,17 @@ object AutomaticConfigTest extends ZIOSpecDefault {
     )
 }
 
-object AutomaticConfigTestUtils {
+object AutomaticConfigSpecUtils {
```

Comment:
For API consistency, I'd like to make this spec shared between Scala 2 and 3. What do you think?

Reply:
Yes, something that was in my radar as well. Agree.